symphony-bdk-http/symphony-bdk-http-api/src/main/java/com/symphony/bdk/http/api/RegularApiClient.java

```diff
@@ -0,0 +1,9 @@
+package com.symphony.bdk.http.api;
+
+/**
+ * Abstract class to be inherited by concrete implementations of ApiClient
+ * which do not override {@link ApiClient#rotate()}
+ * i.e. target base url never changes.
+ */
+public abstract class RegularApiClient implements ApiClient {
```

Comment:
I don't know if having a empty class is something necessary.

Reply:
+1 not sure to really understand the purpose of this class. I'd prefer to keep using `ApiClient` as much as possible in our codebase. 

Especially since `rotate()` is a default method.

Reply:
Idea was to avoid having a LoadBalancingApiClient containing a LoadBalancingApiClient and make sure it contains only a non load balanced ApiClient. 
I can indeed remove the class.